ChangeLog.md

```diff
@@ -24,6 +24,8 @@ See docs/process.md for more on how version tagging works.
   source map : use `-sSOURCE_MAP_PREFIXES=["<old>=<new>"]` with `-gsource-map`.
   Alternatively, you can now embed the sources content into the source map file
   using `-gsource-map=inline`. (#23741)
+- Three settings which have been marked as deprecated for over 12 months were
+  removed: SUPPORT_ERRNO, EXTRA_EXPORTED_RUNTIME_METHODS, DEMANGLE_SUPPORT
```

Comment:
Please add instructions for users here, e.g., 
```
* `EXTRA_EXPORTED_RUNTIME_METHODS`: Use `EXPORTED_RUNTIME_METHODS` instead.
* `DEMANGLE_SUPPORT`: ...
```
Or link somewhere appropriate. Otherwise they might not know where to find that information.